src/PoCs/PopoverFifteenMinutes.tsx

```diff
@@ -0,0 +1,100 @@
+import { Popover, ProgressStep, ProgressStepper, Button, Alert } from "@patternfly/react-core";
+import React, {
+  FunctionComponent,
+  useState,
+  VoidFunctionComponent,
+} from "react";
+
+
+type KafkaStatusPopoverProps = {
+    shouldStartOpen?: boolean;
+  } & KafkaStatusPopoverBodyProps;
+  
+  export const KafkaStatusPopover: FunctionComponent<KafkaStatusPopoverProps> = ({
+    shouldStartOpen = false,
+    currentState,
+    children,
+  }) => {
+    const [isVisible, setIsVisible] = useState(shouldStartOpen);
+  
+    return (
+      <Popover
+        headerContent="Creating instance"
+        bodyContent={<KafkaStatusPopoverBody currentState={currentState} />}
+        isVisible={isVisible}
+        shouldOpen={() => setIsVisible(true)}
+        shouldClose={() => setIsVisible(false)}
+      >
+        {children}
+      </Popover>
+    );
+  };
+  
+  type States = "pending" | "provisioning" | "preparing" | "ready";
+  
+  type KafkaStatusPopoverBodyProps = {
+    currentState: States;
+  };
+  export const KafkaStatusPopoverBody: VoidFunctionComponent<
+    KafkaStatusPopoverBodyProps
+  > = ({ currentState }) => {
+    const order: States[] = ["pending", "provisioning", "preparing", "ready"];
+    const currentStep = order.findIndex((v) => v === currentState);
+    console.log(currentStep);
+    return (
+      <div>
+        <Alert variant="warning" isInline isPlain title="This is taking longer than expected." />
+        <p>In the meantime you can create a service account under the <a href="http://www.google.com/">Connections tab.</a> </p>
```

Comment:
I believe "In the meantime, you can create a service account under the Connections tab." should be "In the meantime, you can create a service account under the Connections tab. If your Kafka instance continues to take longer than expected, open a support case."

Reply:
(Line 47)

Reply:
Thank you fo looking over all of this @kaedward ! 
I think in one of the UX review meetings it was suggested to only link a support case after 30 minutes - I will double check this and if so, I will ensure it is noted in the design doc (it might be out of sync on this)!